usr/local/share/bastille/pkg.sh

```diff
@@ -47,6 +47,10 @@ fi
 
 for _jail in ${JAILS}; do
     info "[${_jail}]:"
-    jexec -l "${_jail}" /usr/sbin/pkg "$@"
+    if [ -f ${_jail}/usr/sbin/pkg ]; then
```

Comment:
This looks odd to me. `${_jail}` should just be name so `${_jail}/usr/sbin/pkg` isn't a full path to evaluate?

Reply:
You're right, I pushed a fix up.